src/main/resources/data/ancientology/recipes/bone_boots.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
The formatting in this file is very weird... Same applies for all of your recipes...